File: DoitMission03/app/src/main/java/com/example/doitmission03/MainActivity.java
package com.example.doitmission03;

import androidx.appcompat.app.AppCompatActivity;

import android.content.res.Resources;
import android.graphics.Bitmap;
import android.graphics.drawable.BitmapDrawable;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import android.widget.ScrollView;

public class MainActivity extends AppCompatActivity {
    ScrollView scrollView1;
    ScrollView scrollView2;
    ImageView imageView1;
    ImageView imageView2;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        scrollView1 = findViewById(R.id.scrollView);
        scrollView2 = findViewById(R.id.scrollView1);
        imageView1 = findViewById(R.id.imageView10);
        imageView2 = findViewById(R.id.imageView11);
        scrollView1.setHorizontalScrollBarEnabled(true);
        scrollView2.setHorizontalScrollBarEnabled(true);

    }
    public void upButton(View v){
        imageView1.setVisibility(View.VISIBLE);
        imageView2.setVisibility(View.INVISIBLE);
    }
    public void downButton(View v){
        imageView2.setVisibility(View.VISIBLE);
        imageView1.setVisibility(View.INVISIBLE);
    }

}

File: HW02_apptive/app/src/main/java/com/example/hw02_apptive/MainActivity.kt
package com.example.hw02_apptive

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import androidx.lifecycle.Observer
import androidx.lifecycle.ViewModelProvider
import androidx.navigation.NavController
import androidx.navigation.Navigation
import androidx.navigation.findNavController

class MainActivity : AppCompatActivity() {

    lateinit var navController:NavController // 네비게이션 컨트롤러

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        val nav_host_fragment : View? = findViewById(R.id.nav_host_fragment)

        // login 프래그먼트 실행
        if (nav_host_fragment != null) {
            navController = nav_host_fragment.findNavController()
        }



    }
}